manifests/dropin_file.pp

```diff
@@ -40,6 +40,9 @@
 # @attr show_diff
 #   Whether to show the diff when updating dropin file
 #
+# @attr notify_service
```

Comment:
I noticed this was wrong and opened https://github.com/camptocamp/puppet-systemd/pull/192

Reply:
#192 is merged. Do you want to rebase?

Reply:
Rebased & fixed. Also regenerated REFERENCE.md